packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -708,6 +709,22 @@ mixin WidgetInspectorService {
   final List<String> _serializeRing = List<String>(20);
   int _serializeRingIndex = 0;
 
+  // A map of the stringified tuple of _Location to the head and tail of
+  // a doubly-linked list of live element instances.
+  final Map<_Location, _ElementLocation> _activeElements = HashMap<_Location, _ElementLocation>.identity();
+
+  // All active locations in a given file. There should only be a handful per
+  // file, making a linear search fast enough.
+  final Map<String, Set<_Location>> _locations = HashMap<String, Set<_Location>>();
+
+  // Fields used to track active instances of the element for faster
+  // invalidations. Together these form a double-linked list of elements,
+  // where every element corresponds to a Widget with an indentical location.
+  // The expando allows adding previous and next pointers to the element clas
+  // without modifying the public API. Unlike a map, it only weakly retains objects.
+  final Expando<Element> _previousElements = Expando<Element>();
```

Comment:
nit: 
`_previousElement`, `_nextElement` as this is an expando for a single next element.